crates/topos-tce-synchronizer/src/checkpoints_collector/mod.rs

```diff
@@ -184,11 +189,14 @@ impl CheckpointSynchronizer {
         diff: HashMap<SubnetId, Vec<ProofOfDelivery>>,
     ) -> Result<Vec<Vec<CertificateId>>, SyncError> {
         let mut certs: HashSet<CertificateId> = HashSet::new();
-        for (_subnet, proofs) in diff {
+        for (subnet, proofs) in diff {
             let len = proofs.len();
             let unverified_certs = self.store.insert_unverified_proofs(proofs)?;
 
-            debug!("Persist {} unverified proofs", len);
+            debug!(
+                "Persist {} unverified proof of delivery for {}",
+                len, subnet
+            );
             certs.extend(&unverified_certs[..]);
```

Comment:
```suggestion
            debug!("Persist {} unverified proofs for {}", proofs.len(), subnet);
```
Doesn't seemt to be used elesewhere?

Reply:
I don't understand the suggestion

Reply:
The suggestion is off, not sure what happened here, maybe some bug in the vscode PR review plugin. Anyway, the suggestion here was to remove the `let len = proofs.len()` line and just use `proofs.len()` in the debug statement. Just a nitpick really.